client/packages/electron/src/electron.ts

```diff
@@ -387,3 +395,107 @@ app.addListener(
     return callback(true);
   }
 );
+
+const isMac = process.platform === 'darwin';
+const appMenu: MenuItemConstructorOptions[] = isMac
+  ? [
+      {
+        label: app.name,
+        submenu: [
+          { role: 'about' },
+          { type: 'separator' },
+          { role: 'services' },
+          { type: 'separator' },
+          { role: 'hide' },
+          { role: 'hideOthers' },
+          { role: 'unhide' },
+          { type: 'separator' },
+          { role: 'quit' },
+        ],
+      },
+    ]
+  : [];
+const fileMenu: MenuItemConstructorOptions = {
+  label: 'File',
+  submenu: [isMac ? { role: 'close' } : { role: 'quit' }],
+};
+const editMenu: MenuItemConstructorOptions = {
+  label: 'Edit',
+  submenu: [
+    { role: 'undo' },
+    { role: 'redo' },
+    { type: 'separator' },
+    { role: 'cut' },
+    { role: 'copy' },
+    { role: 'paste' },
+    ...(isMac
```

Comment:
The original issue just mentions ability to 'clear' data, and it sounds like we don't need it anymore.

Why do we want all of these  ? ('Speech') ? 

Personally I think it's good to have Documentation and maybe File -> Close

Reply:
and maybe about

Reply:
I took the set of defaults which electron implements if you don't specify a menu. Agree though, it can be cut down